### PR TITLE
added kld-loss

### PR DESCRIPTION
**Pull request by mehradans92**

### CODE REVIEW

File: data/kld_loss/kld.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+def kld(true_data, predicted_data):
+  """
+  This function calculates the Kullback–Leibler (KL) divergence loss between predicted data and true data.
```
Comment (hgandhi2411) [inline data/kld_loss/kld.txt:3]: Your variable names are `predicted_data` and `true_data`, should be consistent in the prompt.

**Review by hgandhi2411 — CHANGES_REQUESTED** [review]
I have a comment about variable names in the prompt, all else LGTM

File: data/kld_loss/kld.txt (hunk 1 above)
Comment (mehradans92) [reply data/kld_loss/kld.txt:3]: I don't think the variables name matter. Do they @whitead?

Comment (mehradans92) [reply data/kld_loss/kld.txt:3]: Oh never mind. I see what you mean

**Review by hgandhi2411 — APPROVED** [review]

**Review by geemi725 — APPROVED** [review]

**Pull request merged**